docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,109 @@
-Assessment
-==========
+Introduction
+============
 
-Metrics
--------
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
 
-.. currentmodule:: fairlearn.metrics
+The steps of the assesment are as follows:
+1. Identify harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
+
+1. Identify harms
+-----------------
+
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, high error rates constitute harm in the 
+quality of service.
+
+2. Identify the groups that might be harmed
+-------------------------------------------
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular application. For example, for 
+speech-to-text transcription, groups based on the regional dialect or being a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
+
+3. Quantify harms
+-----------------
+
+Define metrics that quantify harms or benefits:
+
+* In job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are qualified). One possible metric is the false negative rate: fraction of qualified candidates that are screened out.
+
+* In speech-to-text scenario, the harm could be measured by word error rate, number of mistakes in a transcript divided by the overall number of words.
+
+Note that in some cases, harms might not be readily identifiable. If you 
+choose to use a proxy to represent the harm, check the proxy regularly 
+to ensure it remains useful over time.
+
+4. Compare quantified harms across the groups
+---------------------------------------------
+
+The workhorse of fairness assessment are disaggregated metrics, which are 
+metrics evaluated on slices of data. For example, to measure harms due to 
+errors, we would begin by evaluating the errors on each slice of the data that 
+corresponds to a group. If some of the groups are seeing much larger errors 
```

Comment:
I wonder if this is one place where we can raise the issue of what is considered "much larger" and the different approaches to that and their implications (e.g., the issue raised in the [paper](https://arxiv.org/abs/2202.09519) @LeJit mentioned on the call today). This might be a big ask for this PR, though, so we could maybe revisit this in future issues and PRs. @hildeweerts and @LeJit what do you think?

Reply:
I think we could potentially add something that mentions the four-fifths rule in context of U.S. federal employment law, but adds a disclaimer that what constitutes "much larger" is highly context-dependent and a threshold that is applicable in one domain may not translate to a different application.

Reply:
In light of the conversation today, I'm thinking a path forward would be to table this addition until we have added more information about epistemic trespassing to the user guide. Then we could reference that page in this section. @mmadaio @LeJit thoughts?

Reply:
Keeping as is and resolving the comment - thanks!